test/models/report_test.rb

```diff
@@ -0,0 +1,36 @@
+require "test_helper"
+
+class ReportTest < ActionController::TestCase
+  test "valid? returns false when report is too short" do
```

Comment:
Just a naming suggestion, instead of naming a method an a boolean result, we could say `test "is invalid when report is too short" do` decoupling it from the actual code.